vars.tf

```diff
@@ -13,11 +13,17 @@ variable "vpc_cidr" {
   default     = "172.20.0.0/16"
   description = "CIDR range for the VPC?"
   validation {
-    condition     = cidrsubnet(var.vpc_cidr, 3, 5) != ""
+    condition     = cidrsubnet(var.vpc_cidr, 3, 5 + var.subnet_cidrs_offset) != ""
     error_message = "A larger CIDR range must be provided."
   }
 }
 
+variable "subnet_cidrs_offset" {
```

Comment:
A better name for this might be better but I can't manage to come up with something better right now 😄

Reply:
Might even be better to allow passing the CIDR ranges as `var`s ?

Reply:
Thought about that as well. The offset worked for the two cases I had but perhaps a map from string to CIDR with overrides is better? We need one for private and one for public in that case, right?

Reply:
Ok, added public/private_subnet_cidrs instead.